docs/general/administration/hardware-selection.md

```diff
@@ -50,6 +50,13 @@ If you are planning to use a dedicated graphics card (including upgrading an old
 
 Intel Drivers are much easier to install on Linux, with many distributions including them by default. If you do not need CUDA for other applications, it is highly recommended that you stick with Intel Graphics on Linux.
 
+### Servers without GPUs
+
+Not having a GPU is **NOT** recommended for Jellyfin. The following specs are provided for reference only, for software tone-mapping 4K content to SDR:
+
+- CPU: AMD Ryzen 9 5950X, Intel Core i7-12700K or better
```

Comment:
This only applies to Dolby Vision, videos does not require dolby vision reshaping do not need 16core zen3, and the 16core requirement is 60fps playback, so of the goal is 24fps the requirement can also be lower. Also, if users are having 12700K they can just use the integrated graphics.

The requires CPU power differs vastly for use cases and probably should not be documented like this because 4K60 DV tone mapping + transcoding is probably the most demanding task.